### DISCUSSION TITLE
Dynamic content for previews

### DISCUSSION THREAD

**Discussion by IvanSanchez-uam** [cashapp/paparazzi#947] [Q&A] [answered]
Some of the previews have dynamic text like lorem or random numbers values just as an example, when I record the golden screenshots all work properly, then if I then verify the screenshots works fine as well, but I have a problem when:

Let's suppose we only have 2 golden screenshots recorded A and B, now if I add a new preview C in the project

When executing verify task it will fail because C is new
Now I will execute the record command to get the golden screenshot of C
As far as A and B have dinamic values the screenshots will be updated and the idea is just record the new screenshot for C.

To me it makes sense to have dynamic values in my previews, is there a way to tell paparazzi to just record screenshots for new previews?, if not what is the best approach for this scenario?

**Comment by TWiStErRob** [comment, marked as answer]
I think that might be just the question of running (e.g. from [Gradle CLI](https://docs.gradle.org/current/userguide/java_testing.html#simple_name_pattern)) the specific test that you want to refresh. Then the code that would generate a different screenshot for A and B never executes, so it's never re-written.

Alternatively since you're version-controlling golden values (I guess), you can just not commit the modified A and B files; only C, which will be clearly a new file, right?

Btw, how does verify work if the data changes? Do you have a high enough percentage diff that varying values are accepted?
